### PR TITLE
Update README.me to mention run-ldc93s1.sh and be more explicit about GPU requirements

### PR DESCRIPTION
**Pull request by reuben**
I also fixed the part that said we depend on IronPython (what! :P) and made the dependency on Paramiko for the website stuff be optional.

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
I'm wondering which is the most user friendly way of getting users up and running, the notebook or `run-ldc93s1.sh`.

Chris just landed a PR changing the readme stressing the notebook, and your PR is changing it to stress `run-ldc93s1.sh`. Truthfully I'm not sure what the right answer is.

Maybe the GUI is a but more impressive for new users, a bit more flashy? What do you think?

**Comment by reuben** [comment]
Somehow I missed your comment. I think the UI is more flashy, yeah. With the default hyperparameters all matching the requirements for LDC93S1, I'm gonna change the README to stress the notebook and mention the run scripts briefly.

**Comment by kdavis-mozilla** [comment]
@reuben Did you get a change to change the README?

**Comment by reuben** [comment]
I'm traveling with terrible internet and no access to a laptop, so I'll
only be able to get to it next week. Github won't let the edit files on a
mobile browser.

**Comment by reuben** [comment]
I changed the text on rev d4db213 and rebased all of it on top of master.

File: DeepSpeech.ipynb [hunk 1]
```diff
@@ -101,7 +101,11 @@
     "import importlib\n",
     "ds_importer_module = importlib.import_module('util.importers.%s' % ds_importer)\n",
     "\n",
-    "from util.website import maybe_publish\n",
+    "try:\n",
```
Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Why is this part of this PR? Shouldn't this be in its own PR as it has nothing to do with the `README.me`

Comment (reuben) [inline DeepSpeech.ipynb]: The context of this PR was "make the repository friendlier to external contributors" so I bundled this in because it was in the back of my mind when I was touching the external dependencies list. I can split it out if you prefer.

Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Could you split it out if it's not too much work.

**Comment by reuben** [comment]
I removed that commit from this PR.
